octodns_gcore/__init__.py

```diff
@@ -12,10 +12,9 @@
 from octodns import __VERSION__ as octodns_version
 from octodns.provider import ProviderException
 from octodns.provider.base import BaseProvider
-from octodns.record import GeoCodes, Record
+from octodns.record import GeoCodes, Record, Update
 
-# TODO: remove __VERSION__ with the next major version release
-__version__ = __VERSION__ = '0.0.4'
+__VERSION__ = '0.0.5'
 
```

Comment:
```suggestion
# TODO: remove __VERSION__ with the next major version release
__version__ = __VERSION__ = '0.0.4'
```

Version bumps happen in dedicated release PRs rather than in feature branches/PRs.